src/User_Command.php

```diff
@@ -280,8 +280,12 @@ public function delete( $args, $assoc_args ) {
 			WP_CLI::error( 'Reassigning content to a different user is not supported on multisite.' );
 		}
 
+		$is_reassign_valid = ( false === get_userdata( $reassign ) ) ? false : true;
```

Comment:
```suggestion
		$is_reassign_valid = ( $reassign && false === get_userdata( $reassign ) ) ? false : true;
```

Let's avoid the DB hit unless `$reassign` is provided.